Add documentation for databricks column configs (#7479)
dbt-labs/docs.getdbt.com: website/docs/docs/core/connect-data-platform/databricks-setup.md

> You can use either token-based authentication or OAuth client-based authentication to connect to Databricks. Refer to the following examples for more info on how to configure your profile for each type of authentication.

Reviewer: ```suggestion
You can use either token-based authentication or OAuth client-based authentication to connect to Databricks. Refer to the following examples for more on how to configure your profile for each type of authentication.
```

> The default OAuth app for dbt-databricks is auto-enabled in every account with expected settings, you can find it in [Account Console](https://accounts.cloud.databricks.com) > [Settings](https://accounts.cloud.databricks.com/settings) > [App Connections](https://accounts.cloud.databricks.com/settings/app-integrations) > dbt adapter for Databricks. If you cannot find it you may have disabled dbt in your account, please refer to this [guide](https://docs.databricks.com/en/integrations/enable-disable-oauth.html) to re-enable dbt as OAuth app.

Reviewer: Edits to be more specific:
```suggestion
The default OAuth app for dbt-databricks is auto-enabled in every account with expected settings.  You can find the adapter app in [Account Console](https://accounts.cloud.databricks.com) > [Settings](https://accounts.cloud.databricks.com/settings) > [App Connections](https://accounts.cloud.databricks.com/settings/app-integrations) > dbt adapter for Databricks. If you cannot find the adapter app, dbt may be disabled in your account, please refer to this [guide](https://docs.databricks.com/en/integrations/enable-disable-oauth.html) to re-enable dbt-databricks as an OAuth app.
```
